src/locale/locales/tr/messages.po

```diff
@@ -1368,7 +1368,7 @@ msgstr "E-postanızı Değiştirin"
 #: src/view/shell/desktop/LeftNav.tsx:329
 #: src/view/shell/Drawer.tsx:446
 msgid "Chat"
-msgstr ""
+msgstr "Mesajlar"
```

Comment:
For the native turkish speakers:
Sol navigasyon bölümünde gösterilen Chat bölümünü twitter ile aynı olması için Mesajlar şeklinde çevirdim.